File: java/remoting/main/com/taobao/top/link/remoting/CrossLanguageSerializationFactory.java
package com.taobao.top.link.remoting;

public class CrossLanguageSerializationFactory extends DefaultSerializationFactory {
	private Serializer jsonSerializer = new CrossLanguageJsonSerializer();

	@Override
	public Serializer get(Object format) {
		if (format != null && "json".equals(format))
			return this.jsonSerializer;
		return super.get(format);
	}
}

File: csharp/src/Top.Link/Remoting/Serialization/Json/CrossLanguageSerializationFactory.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Taobao.Top.Link.Remoting.Serialization.Json
{
    public class CrossLanguageSerializationFactory : ISerializationFactory
    {
        private CrossLanguageJsonSerializer _jsonSerializer = new CrossLanguageJsonSerializer();
        public ISerializer Get(object format)
        {
            return this._jsonSerializer;
        }
    }
}
